Fix-Website/app/app.js

```diff
@@ -34,25 +32,44 @@
     app.directive("infoZone", function () {
         return {
             restrict: 'E',
-            templateUrl: "Templates/info.html",
-            controller: function () {
-                this.form = false;
-                this.clickProduct = function () {
-
-                    this.form = !this.form;
-                };
-            },
-            controllerAs: 'info'
+            templateUrl: "Templates/info.txt"
         };
     });
 
-    app.controller('ReferenceController', ['$scope', function ($scope) {
+    /**
+     * Controller for setting up the scope.
+     */
+    app.controller('ReferenceController', ['$scope','$http', function ($scope, $http) {
         $scope.frame = 1;
```

Comment:
You should avoid defining things on the scope.  This clutters the scope object.  You should either make services or define things on controllers.